lib/rubocop/cop/rails/inquiry.rb

```diff
@@ -26,7 +26,11 @@ class Inquiry < Cop
         MSG = "Prefer Ruby's comparison operators over Active Support's `inquiry`."
 
         def on_send(node)
-          add_offense(node, location: :selector) if node.method?(:inquiry) && node.arguments.empty?
+          return unless node.method?(:inquiry) && node.arguments.empty?
+          return unless (receiver = node.receiver)
```

Comment:
How about adding a test that covers `inquiry` with no receiver, since that's what this line is for?

Reply:
Sure! I added the test.